rollup/conf/config.json

```
@@ -19,7 +19,8 @@
         "min_gas_price": 0,
         "gas_price_diff": 50000,
         "l1_base_fee_weight": 0.132,
-        "l1_blob_base_fee_weight":  0.145
+        "l1_blob_base_fee_weight":  0.145,
+        "gas_price_oracle_update_window": 5
```

We should pick a more intuitive name, e.g.

```suggestion
        "commit_batch_timeout_window_minutes": 5
```